conf/global.json

```diff
@@ -43,12 +43,12 @@
   },
   "s3_access_logging": {
     "create_bucket": true,
-    "logging_bucket": "PREFIX_GOES_HERE.streamalert.s3-logging"
+    "logging_bucket": "PREFIX_GOES_HERE-streamalert-s3-logging"
```

Comment:
Random and this may be handled already; according to the Transfer acceleration docs, the S3 bucket names need to confirm to DNS naming conventions, so underscores can't be used right?

Reply:
yes underscores are not allowed - do you see somewhere that they're being used?

Reply:
I think we're already compliant with that restriction

Reply:
Ah
I'm stupid
`PREFIX_GOES_HERE` gets replaced

Reply:
oh yeah I also want to fix that .. https://github.com/airbnb/streamalert/issues/1086

Reply:
We probably also need to sanity check the prefix, LOL. Maybe someone use period in the prefix then it breaks the S3 Transfer acceleration support.

Reply:
I think we already have something that checks for invalid characters in the prefix (underscores in particular). I'll check for it and updates accordingly

Reply:
this is now configurable